orders/admin.py

```diff
@@ -1,3 +1,11 @@
 from django.contrib import admin
+from orders.models import Order, LineItem
 
-# Register your models here.
+class OrderAdmin(admin.ModelAdmin):
+    pass  
+
+class LineItemAdmin(admin.ModelAdmin):
+    pass
+
```

Comment:
Please use decorator instead this approach, when you use decorator, it's give more flexible code